src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -267,29 +204,13 @@ public static ColumnFilter selection(RegularAndStaticColumns columns)
      * A filter that fetches all columns for the provided table, but returns
      * only the queried ones.
      */
+    @VisibleForTesting
```

Comment:
i could not find anywhere this method was used but the tests…

Reply:
Me neither, +1